code/smoothie.js

```diff
@@ -0,0 +1,314 @@
+  amountName: 'raspberries',
+  image: './images/raspberry.jpg',
+  tastesGoodWith: 'Milk and oats'
+}
+
+const lemon = {
+  name: 'Lemon',
+  color: 'yellow',
+  category: 'sour',
+  amount: 2,
+  amountName: 'lemons',
+  image: './images/lemon.jpg',
+  tastesGoodWith: 'Coconut water'
+}
+
+// ------------------ Mixer ------------------
+const almondMilk = {
+  name: 'Almond milk',
+  image: './images/almondmilk.jpg'
+}
+
+const yoghurt = {
+  name: 'Yoghurt',
+  image: './images/yoghurt.jpg'
+}
+
+const coconutWater = {
+  name: 'Coconut water',
+  image: './images/coconutwater.jpg'
+}
+
+const milkOats = {
+  name: 'Milk and oats',
+  image: './images/oat.jpg'
+}
+
+// ------------------ Superboosters ------------------
+const spirulina = {
+  name: 'Spirulina',
+  amount: 'about 1 tbsp',
+  image: './images/spirulina.jpg'
+}
+
+const ginger = {
+  name: 'Ginger',
+  amount: 'about 2 cm',
+  image: './images/ginger.jpg'
+}
+
+const kale = {
+  name: 'Kale',
+  amount: 'about a fist',
+  image: './images/kale.jpg'
+}
+
+const chiaSeeds = {
+  name: 'Chia seeds',
+  amount: 'about 2 tbsp',
+  image: './images/chia.jpg'
+}
+
+// Arrays made with all ingredients, separated into 3 groups
+const mainFruits = [
+  apple,
+  banana,
+  mango,
+  orange,
+  raspberry,
+  lemon
+]
+
+const mixers = [
+  almondMilk,
+  yoghurt,
+  coconutWater,
+  milkOats
+]
+
+const superBoosters = [
+  spirulina,
+  ginger,
+  kale,
+  chiaSeeds
+]
+
+// Array for favorites (Objects will be added when favorite is selected in browser)
+let mainFruitsPlusFavorite = [
+  apple,
+  banana,
+  mango,
+  orange,
+  raspberry,
+  lemon
+]
+
+// Object that will define all ingredients currently selected in the smoothie,
+// defined in the global scope
+let smoothie = {
+  mainFruit: undefined,
+  mixer: undefined,
+  superBooster: undefined
+}
+
+// Defining the favoriteFruit-variable in the global scope
+let favoriteFruit = undefined
+
+// Select a favorite fruit to make it appear more frequently in the mix
+const selectFavorite = (value) => {
+  event.preventDefault()
+  favoriteFruit = value.replace(/(^"|"$)/g, '')
+
+
+  // This is how I would solve the "Shuffle Favorite Weapon"-problem:
+  // (I would connect the function to an onclick event on a button, 
+  // and run that with an if-statement similar to the one below)
+
+  // const randomFavorite = () => {
+  //   favoriteFruit = randomSelector(mainFruits).name
+  //   console.log("Hello", favoriteFruit)
+  // }
+
+
+  // If a favorite is selected, remove the extra favorite-objects in array
+  const removeFavorites = () => {
+    if (mainFruitsPlusFavorite.length > 6) {
+      mainFruitsPlusFavorite.splice(-3, 3)
+    }
+  }
+
+  // If favorite is selected in browser, add matching object from mainFruit-array x 3 
+  // to a new array which will later be used to randomize fruit 
+  if (favoriteFruit == "Apple") {
+    removeFavorites()
+    mainFruitsPlusFavorite.push(mainFruits[0], mainFruits[0], mainFruits[0])
```

Comment:
Maybe there's a way to do this only using the index? 
```javascript
mainFruitsPlusFavorite.push(mainFruits[index], mainFruits[index], mainFruits[index])
```

Then you don't need the `if`s